src/core/exception.d

```diff
@@ -135,7 +135,7 @@ class FinalizeError : Error
 
     @safe override string toString() const
     {
-        return "An exception was thrown while finalizing an instance of class " ~ info.name;
+        return "An exception was thrown while finalizing an instance of " ~ info.toString();
```

Comment:
How does that even work? It allocates. Should use a sink delegate. Anyhow, not part of this pull.